bundles/org.openhab.binding.plex/README.md

```diff
@@ -0,0 +1,172 @@
+
+## Thing Configuration
+
+The PLEX Server needs to be configured first. The hostname of the PLEX server is mandatory and the either the PLEX token (recommended) or the username/password of the PLEX server.
+
+The find the PLEX token please follow the instructions from the PLEX support forum: 
+
+1. Sign in to your Plex account in Plex Web App
+2. Browse to a library item and view the XML for it
+3. Look in the URL and find the token as the X-Plex-Token value
+
+### `PLEX Server` Thing Configuration
+
+| Name        | Type    | Description                                                                                                                                                                                               | Default | Required | Advanced |
+|-------------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|----------|---------|
+| host        | text    | PLEX host name or IP address                                                                                                                                                                              | N/A     | yes      | no      |
+| portNumber  | integer | Port Number (leave blank if PLEX installed on default port)                                                                                                                                               | 32400   | no       | no      |
+| refreshRate | integer | Interval in seconds at which PLEX server status is polled                                                                                                                                                 | 5       | no       | no      |
+| username    | text    | If you're using Plex Home you need to supply the username and password of your Plex account here. If you don't want to enter your credentials you can also directly set your account token below instead. | N/A     | no       | no      |
+| password    | text    | If you're using Plex Home you need to supply the username and password of your Plex account here. If you don't want to enter your credentials you can also directly set your account token below instead. | N/A     | no       | no      |
+| token       | text    | The authentication token when username/password is left blank                                                                                                                                             | N/A     | no       | no      |
+
+### `PLEX player` Thing Configuration
+
+You can add multiple PLEX players. You can choose to find the player by autodiscovery or add them manually.
+
+#### Autodiscovery
+
+Turn on the player you want to add and *play* some media on it. Navigate to `/settings/things/add/plex` and press the `SCAN` button.
```

Comment:
MainUI specific documentation should be avoided. You could simply say, that the auto discovery can be started.